nucliadb/nucliadb/tests/fixtures.py

```diff
@@ -186,7 +186,7 @@ async def nucliadb(
 async def nucliadb_reader(nucliadb: Settings):
     async with AsyncClient(
         headers={"X-NUCLIADB-ROLES": "READER"},
-        base_url=f"http://localhost:{nucliadb.http_port}/{API_PREFIX}/v1",
```

Comment:
These are the actual changes in the fixtures to facilitate working with endpoints on different versions of the api